pydap/handlers/pcic/__init__.py

```diff
@@ -182,6 +182,10 @@ def get_full_query(self, stn_id, sesh):
            :type stn_id: int or str
            :param sesh: an sqlalchemy session
         '''
+	
+        if not self.get_vars(stn_id, sesh):
+            return "SELECT obs_time FROM obs_raw WHERE 1 = 0"
```

Comment:
It's kind of six of one and half a dozen of the other, but I'd probably write this as ``WHERE NULL`` rather than ``WHERE 1 = 0``. End result is that you need a query that will return a table with column `obs_time` (as required [here](https://github.com/pacificclimate/pydap.handlers.pcic/blob/5f5a13b8fae95a683409da60f422611f94a4f223/pydap/handlers/pcic/__init__.py#L115) and no rows.

I'd recommend adding a bit of an explanation to the docstring of ``get_full_query()``, because at first glance, it's not exactly clear *why* this works (though it becomes clear if you dig into implementation details in other functions). It's probably worth writing something like "This function needs to return a string for an SQL query that would return at least one column (obs_raw) and optionally multiple columns for each available variable."